ui/perfherder/alerts/Pagination.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Row, Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+class Paging extends React.Component {
+  constructor(props) {
+    super(props);
+    this.validated = this.props.validated;
+    this.state = {
+      page: this.validated.page ? parseInt(this.validated.page, 10) : 1,
```

Comment:
You could set the `page` prop type as not required, with a default value of 1 instead of reusing this ternary logic.